### ISSUE TITLE
count not aggregating correctly

### ISSUE THREAD

**Issue by quoll** [threatgrid/asami#135]
Seeing duplicates with `count` operations:

```clojure
(asami/q '[:find ?ip (count ?sighting)
           :where
           [?sighting :type "sighting"]
           [?sighting :targets ?targets]
           [?targets :tg/contains ?target]
           [?target :type "endpoint"]
           [?target :observables ?obss]
           [?obss :tg/contains ?obs]
           [?obs :type "ip"]
           [?obs :value ?ip]]
         (:db-uri engine))
```
![Screen Shot 2021-04-19 at 4 13 58 PM](https://user-images.githubusercontent.com/358875/115305086-ce364800-a133-11eb-8d6f-fe6297b18ccd.png)

**Comment by quoll** [comment]
This looks like it could be the subquery selection. Check out what out queries and subqueries were created

**Comment by quoll** [comment]
Addressed in PR #138
